core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -46,13 +47,16 @@ class ModOptions : IHasUniques {
     var author = ""
     var modSize = 0
  
+    @Deprecated("As of 3.18.15")
     val maxXPfromBarbarians = 30
 
     override var uniques = ArrayList<String>()
     // If this is delegated with "by lazy", the mod download process crashes and burns
     override var uniqueObjects: List<Unique> = listOf()
     override fun getUniqueTarget() = UniqueTarget.ModOptions
 
+    val constants = ModConstants()
```

Comment:
Also.

Reply:
That "crashes and burns" comment still there? It's wrong. A by lazy and deserialization mandates a transient annotation like everywhere else, that's all...